docs/benchmarks/web-search.md

```diff
@@ -12,47 +12,65 @@ The Web Search benchmark relies on the [Apache Solr][apachesolr] search engine f
 ### Dockerfiles ###
 
 Supported tags and their respective `Dockerfile` links:
-
+- [`index`][indexdocker] This builds an image that crawls a set of given websites to generate an index.
+- [`dataset`][datasetdocker] This downloads an already generated index and mounts it to be used by the server.
 - [`server`][serverdocker] This builds an image for the Apache Solr index nodes. You may spawn several nodes.
 - [`client`][clientdocker] This builds an image with the client node. The client is used to start the benchmark and query the index nodes.
 
-These images are automatically built using the mentioned Dockerfiles available on [`https://github.com/parsa-epfl/cloudsuite/tree/master/benchmarks/web-search`][repo].
+These images are automatically built using the mentioned Dockerfiles available [`here`][repo].
 
 ### Starting the dataset ###
 
-Downloads the dataset index
+The following command downloads and mounts the dataset index:
 
-	$ docker run cloudsuite/web-search:dataset
+  $ docker run --name web_search_dataset cloudsuite/web-search:dataset
```

Comment:
Can you format all the terminal commands using the way below?

```
$ docker run --name web_search_dataset cloudsuite/web-search:dataset
```

Just using three \` to create a code block.